core/PRP/Animation/plLeafController.cpp

```diff
@@ -441,7 +441,7 @@ void plLeafController::setKeys(const std::vector<hsKeyFrame*>& keys, unsigned in
     DeallocKeys();
     AllocKeys(keys.size(), type);
     for (size_t i=0; i<keys.size(); i++)
-        fKeys[i] = keys[i];
+        *fKeys[i] = *keys[i];
```

Comment:
I think this is mirroring Cyan's implementation somewhat literally. IMO copying the keys is the correct solution in this case--the keys could be coming from another controller, adding the gotcha of one controller stealing another controller's owned keys. There's also the fPyOwned gotcha where we have a TypeError raised and some keys are leaked as a result.